<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xCWdgt</class>
 <widget class="QWidget" name="xCWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xCWdgt</string>
  </property>
  <widget class="QFrame" name="pSettingsPanel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>485</y>
     <width>790</width>
     <height>110</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>79</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="8" rowspan="2">
      <widget class="QSlider" name="pZoomSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QSpinBox" name="pBinningSB">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="2" colspan="2">
      <widget class="QSpinBox" name="pCameraHeightSB">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="pCamWidthLab">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>width</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="7" rowspan="2">
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>zoom</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="9" rowspan="2">
      <widget class="QSpinBox" name="pZoomSB">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QLabel" name="pCamHeightLab">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>height</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>binning</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7" rowspan="2">
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>gain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QSlider" name="pCameraHeightSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1020</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="8" rowspan="2">
      <widget class="QSlider" name="pGainSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum">
        <number>190</number>
       </property>
       <property name="maximum">
        <number>1023</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="9" rowspan="2">
      <widget class="QSpinBox" name="pGainSB">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>190</number>
       </property>
       <property name="maximum">
        <number>1023</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QSpinBox" name="pCameraWidthSB">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1920</number>
       </property>
       <property name="value">
        <number>600</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QSlider" name="pCameraWidthSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1920</number>
       </property>
       <property name="value">
        <number>600</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="5" rowspan="2" colspan="2">
      <widget class="QDoubleSpinBox" name="pFrameRateDSB">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>fps</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4" rowspan="2">
      <widget class="QSlider" name="pFrameRateSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3" rowspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>frame rate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2">
      <widget class="QDoubleSpinBox" name="pExposureDSB">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QSlider" name="pExposureSlider">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>exposure</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="pCamImgGV">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>790</width>
     <height>590</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QFrame" name="pStatusPanel">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>5</y>
     <width>260</width>
     <height>481</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>79</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>52</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>52</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>messages</string>
    </property>
   </widget>
   <widget class="QLabel" name="pStatusLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>not connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="pInfoLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>i1
i2
i3
i4</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pMSGBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pStatusPixLab">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>9</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="xCam_resources.qrc">:/images/traffic_light_empty.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="actionconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="xCam_resources.qrc">
     <normaloff>:/images/connections.png</normaloff>:/images/connections.png</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actionstart_stop_acquiring">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="xCam_resources.qrc">
     <normaloff>:/images/recording.png</normaloff>:/images/recording.png</iconset>
   </property>
   <property name="text">
    <string>start/stop acquiring</string>
   </property>
  </action>
  <action name="actionsettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="xCam_resources.qrc">
     <normaloff>:/images/adjust.png</normaloff>:/images/adjust.png</iconset>
   </property>
   <property name="text">
    <string>settings panel</string>
   </property>
  </action>
  <action name="actioninfo_panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="xCam_resources.qrc">
     <normaloff>:/images/messages.png</normaloff>:/images/messages.png</iconset>
   </property>
   <property name="text">
    <string>info panel</string>
   </property>
  </action>
  <action name="actionset_output_stream">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="xCam_resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>set output stream </string>
   </property>
  </action>
  <zorder>pCamImgGV</zorder>
  <zorder>pSettingsPanel</zorder>
  <zorder>pStatusPanel</zorder>
 </widget>
 <resources>
  <include location="xCam_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pZoomSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pZoomSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pZoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pZoomSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pGainSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pGainSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pGainSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pGainSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>724</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pCameraHeightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pCameraHeightSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pCameraHeightSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pCameraHeightSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pCameraWidthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pCameraWidthSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pCameraWidthSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pCameraWidthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
